plugins/de.ovgu.featureide.fm.attributes/src/de/ovgu/featureide/fm/attributes/view/FeatureAttributeView.java

```diff
@@ -408,7 +410,7 @@ private void createColumns() {
 
 		// Value
 		final TreeViewerColumn colValue = new TreeViewerColumn(treeViewer, SWT.NONE);
-		colValue.getColumn().setWidth(200);
+		colValue.getColumn().setWidth(400);
```

Comment:
a final int would look better, no?

Reply:
I renounced the idea to use a constant member as the width of the column is not really subject of change for future enhancement and also it is not an important property of the view.